book/src/accounts_and_groups.md

```diff
@@ -86,8 +86,8 @@ kanidm person create demo_user "Demonstration User" --name idm_admin
 kanidm person get demo_user --name idm_admin
 
 kanidm group create demo_group --name idm_admin
-kanidm group add-members demo_group demo_user --name idm_admin
-kanidm group list-members demo_group --name idm_admin
+kanidm group add_members demo_group demo_user --name idm_admin
+kanidm group list_members demo_group --name idm_admin
```

Comment:
The current commands are `add-members` and `list-members`.

```shell
cargo run --bin kanidm group --help
kanidm-group
Actions to manage groups

USAGE:
    kanidm group <SUBCOMMAND>

OPTIONS:
    -h, --help    Print help information

SUBCOMMANDS:
    add-members       Add new members to a group
    create            Create a new group
    delete            Delete a group
    get               View a specific group
    help              Print this message or the help of the given subcommand(s)
    list              List all groups
    list-members      List the members of a group

Reply:
Ah, sorry, I didn't realize I was working in master. 

Checking the branch 1.1.0-alpha.11 for the last stable I see that the docs are correct for the code, but the published docs:

https://kanidm.github.io/kanidm/stable/
https://kanidm.github.io/kanidm/v1.1.0-alpha.11/

Do not correspond to the docs in the branch, but to some time in master.

But that is a different problem. I will close the PR as it is completely wrong. Sorry for the inconvenience.

Reply:
No worries, I think there was a short time of overlap between the changes and docs-things are a bit messy 😢 Sorry for the confusion!

Reply:
Thanks @itorres sorry for the confusion. I think we need to clean up some docs things :)